tests/compiletest.rs

```diff
@@ -190,7 +190,8 @@ fn run_pass_miri_noopt() {
 
 #[test]
```

Comment:
Maybe use `#[ignore]` here instead of making the content a commit.

Reply:
Ah, good catch! I will try to remember this next time. I did not know about this attribute.